src/classes/tmConstants.cls

```diff
@@ -0,0 +1,95 @@
+    public static Set<Id> INTERNAL_ACCOUNTS {get { return tmDaoSettings.internalAccounts; } private set;}
+    public static String PRICEBOOK_CLAUSE { get {return 'PriceBookLikeClause';} private set;}
+    public static Boolean CAN_USE_TEMPLATES { get { return tmTrialManagerTemplate__c.SObjectType.getDescribe().isAccessible(); } private set; }
+    public static Boolean TEMPLATES_ENABLED { get { return (tmSettings__c.getInstance('tmTemplatesEnabled') != null && tmSettings__c.getInstance('tmTemplatesEnabled').Value__c.equalsIgnoreCase('true')); } private set; }
+
+    //Labels
+    public static String ERROR_ACCOUNT_TFRID_MISSING {get {return Label.tmAccountTFRRequired;} private set;}
+    public static String ERROR_ACCOUNT_TENANT_MISSING {get {return Label.tmAccountTenantRequired;} private set;}
+    public static String ERROR_TENANT_MISSING {get {return Label.tmTenantRequired;} private set;}
+    public static String ERROR_INITIALIZE_TRIAL_FEATURE_REQUEST {get {return Label.tmTFRInitializeFailed;} private set;}
+    public static String ERROR_SUBMITTING_FOR_PROVISIONING {get {return Label.tmTFROrderCreationFailed;} private set;}
+    public static String ERROR_SAVING_TRIAL_FEATURE_REQUEST {get {return Label.tmTFRSaveFailed;} private set;}
+    public static String ERROR_SUBMITTING_FOR_APPROVAL {get {return Label.tmTFRSubmitForApprovalFailed;} private set;}
+    public static String ERROR_ORG_CHANGED {get {return Label.tmOrgChanged;} private set;}
+    public static String ERROR_TFR_DELETE_LABEL {get {return Label.tmTFRDeleteMsg;} private set;}
+    public static String ERROR_TFR_CANNOT_BE_EDITED {get {return Label.tmCannotEditMsg;} private set;}
+    public static String ERROR_INVALID_BILLING_ADDRESS {get { return Label.tmInvalidBillingAddress; } private set; }
+    public static String ERROR_INTERNAL_ACCOUNT {get { return Label.tmInternalAccount; } private set; }
+    public static String ERROR_INVALID_EDITION {get { return Label.tmInvalidEdition; } private set; }
+    public static String ERROR_INVALID_ISO_COUNTRY_CODE {get { return Label.tmInvalidISOCountryCode; } private set; }
+    public static String ERROR_MISSING_TRIAL_FEATURE_REQUEST_ITEMS {get { return Label.tmMissingTFRItems; } private set; }
+    public static String ERROR_UPGRADE_TFR_CORE_PRODUCT_COUNT {get { return Label.tmUpgradeTFRCoreProductCount; } private set; }
+
+     public static Map<Integer, String> EDITION_ORDER_MAP {
+        get {
+            if (Edition_ORDER_MAP == null)
+                EDITION_ORDER_MAP = new Map<Integer, String> { 1 => 'Group',
+                                                              2 => 'Professional',
+                                                              3 => 'Enterprise',
+                                                              4 => 'Unlimited',
+                                                              5 => 'Performance' };
+            return EDITION_ORDER_MAP;
+           }
+        private set;
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.